tinywallet/tinywallet/app.py

```diff
@@ -233,13 +240,14 @@ def run():
 
     def getPassword(self, f, prompt="Password"):
         """
-        Calls the provided function with a user-provided password string as its
-        first argument. Any additional arguments provided to getPassword are
-        appended as-is to the password argument.
+        Calls the provided function with a user-provided prompt string as its
+        first argument.
```

Comment:
We don't pass the `prompt` to the provided function, just the password entered on the password screen.